cmd/kubeapps-apis/plugins/helm/packages/v1alpha1/server_test.go

```diff
@@ -1371,9 +1372,10 @@ func TestGetAvailablePackageVersions(t *testing.T) {
 
 func TestPackageAppVersionsSummary(t *testing.T) {
 	testCases := []struct {
-		name            string
-		chart_versions  []models.ChartVersion
-		version_summary []*corev1.PackageAppVersion
+		name                    string
+		chart_versions          []models.ChartVersion
+		version_summary         []*corev1.PackageAppVersion
+		exp_versions_in_summary VersionsInSummary
```

Comment:
Just wondering what the "exp_" is for here? This one isn't an expected value, but rather an input value? The expected value is the resulting version summary? Fine either way, just confused me reading it.